src/georaster-layer-for-leaflet.ts

```diff
@@ -321,7 +324,23 @@ const GeoRasterLayer: (new (options: GeoRasterLayerOptions) => any) & typeof L.C
     // note that we aren't setting the tile height or width here
     // drawTile dynamically sets the width and padding based on
     // how much the georaster takes up the tile area
-    this.drawTile({ tile, coords, context, done });
+    const coordsKey = this._tileCoordsToKey(coords);
+    const key = `${coordsKey}:${this.options.resolution}`;
```

Comment:
this looks awesome. one minor thing.  do we need to update this line to use the resolution for the requested zoom level, now that https://github.com/GeoTIFF/georaster-layer-for-leaflet/pull/130 was merged?

Reply:
Yes, fixed now